.circleci/config.yml

```diff
@@ -98,6 +98,12 @@ workflows:
             - js_build
             - build_server_pdfs
             - build_api_docs
+      - minimize-images:
+          requires:
+            - build              
+          filters: 
+            branches:
+              only: master 
```

Comment:
we should probably run this for preview branches as well to help catch if an image is breaking.

Reply:
Good call on this, will be addressed

Reply:
I'm not against this! Good call